slp2mp4/User/Config/Dolphin.ini

```diff
@@ -31,7 +31,7 @@ DisableTooltips = False
 [Display]
 FullscreenResolution = Auto
 Fullscreen = False
-RenderToMain = False
+RenderToMain = True
```

Comment:
It got annoying spawning more windows than necessary. This combines the video window with the main dolphin window.

Reply:
Ok, I think I will make a PR after this to check if the system is windows and then apply a patch to set RenderToMain to true.